content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,567 @@
+
+1.  Create factory.go file and define the unique string to identify your
+    connector as a global constant.
+
+    ```go
+    const (
+        defaultVal = "request.n"
+        // this is the name used to refer to the connector in the config.yaml
+        typeStr = "example"
+    )
+    ```
+
+2.  Create the default configuration function. This is how you choose to
+    initialize your connector object with default values.
+
+    ```go
+    func createDefaultConfig() component.Config {
+        return &Config{
+            AttributeName: defaultVal,
+        }
+    }
+    ```
+
+3.  Define the type of the connector you will work with. This will be passed as
+    a factory option. Usually, a connector connects two pipelines of different
+    types. We have to define the type of the exported end of the connector and
+    the receiver end of the connector.
+
+    ```go
+    // createTracesToMetricsConnector defines the consumer type of the connector
+    // We want to consume traces and export metrics, therefore, define nextConsumer as metrics, since consumer is the next component in the pipeline
+    func createTracesToMetricsConnector(ctx context.Context, params connector.CreateSettings, cfg component.Config, nextConsumer consumer.Metrics) (connector.Traces, error) {
+        c, err := newConnector(params.Logger, cfg)
+        if err != nil {
+            return nil, err
+        }
+        c.metricsConsumer = nextConsumer
+        return c, nil
+    }
+    ```
+
+    `createTracesToMetricsConnector` is a function that further initializes the
+    connector component by defining its consumer component, or the next
+    component to ingest the data after the connector transmits the data.
+
+    #### Parameters for the `createTracesToMetricsConnector`:
+
+    - `context.Context`: the reference to the collector’s `context.Context` so
+      your trace receiver can properly manage its execution context.
+    - `connector.CreateSettings`: the reference to some of the collector’s
+      settings under which your receiver is created.
+    - `component.Config`: the reference for the receiver config settings passed
+      by the collector to the factory so it can properly read its settings from
+      the collector config.
+    - `consumer.Metrics`: the reference to the next consumer type in the
+      pipeline, which is where received traces will go. This can be a processor,
+      exporter or another connector.
+
+4.  Write a `NewFactory` function that instantiates your custom factory for your
+    connector(component).
+    ```go
+    // NewFactory creates a factory for example connector.
+    func NewFactory() connector.Factory {
+        // OpenTelemetry connector factory to make a factory for connectors
+        return connector.NewFactory(
+        typeStr,
+        createDefaultConfig,
+        connector.WithTracesToMetrics(createTracesToMetricsConnector, component.StabilityLevelAlpha))
```

Comment:
It's possible for connectors to support multiple ordered combinations of data types. Given that this is an introduction, it may not be necessary to present this, but it might be worth mentioning somewhere so that readers understand it's not a general limitation.